great_expectations/rule_based_profiler/types/data_assistant_result/plot_components.py

```diff
@@ -1,20 +1,23 @@
 from dataclasses import dataclass
-from typing import Optional, Union
+from typing import List, Optional, Union
 
 import altair as alt
 
 
 @dataclass(frozen=True)
 class PlotComponent:
-    name: str
-    alt_type: alt.StandardType
+    name: Optional[str]
+    alt_type: Optional[alt.StandardType]
```

Comment:
Should we have default values of `None` or no?

Reply:
Yes, and that means everything under `PlotComponent` will be that way. That is fine because there are a ton of cases where I need to turn these attributes on/off. That will be used extensively in a follow-on cleanup PR.